src/main/java/net/datafaker/idnumbers/LatinLetters.java

```diff
@@ -0,0 +1,21 @@
+package net.datafaker.idnumbers;
+
+import java.util.Arrays;
+
+public class LatinLetters {
+
+    private static final char[] VOWELS = {'A', 'E', 'I', 'O', 'U', 'Y'};
+
+    static boolean isConsonant(int c) {
+        return isConsonant((char) c);
+    }
+
+    static boolean isConsonant(char c) {
+        return Arrays.binarySearch(VOWELS, c) < 0;
+    }
```

Comment:
Is this actually faster than using a `List` (`List.of`) and `.contains()`?

Reply:
@kingthorin Thanks, fixed. Now using `Set`.